.github/workflows/ci.yaml

```diff
@@ -85,32 +85,15 @@ jobs:
       - name: Cache Rust artifacts
         uses: actions/cache@v2
         if: |
-          matrix.cmd == 'run-cargo-clippy' || matrix.cmd == 'run-functions-examples --application-variant=rust'
+          matrix.cmd == 'run-cargo-clippy' || matrix.cmd == 'run-examples --application-variant=rust' || matrix.cmd == 'run-examples --application-variant=cpp' || matrix.cmd == 'run-functions-examples --application-variant=rust'
         with:
           path: |
             ./cargo-cache/bin
             ./cargo-cache/registry/index
             ./cargo-cache/registry/cache
             ./cargo-cache/git/db
             ./sccache-cache
-            ./examples/abitest/module_0/rust/target
```

Comment:
IIUC correctly `abitest/module_0&1` have their own workspace and do not build into `./target`. As we don't keep them here, this means they won't be cached, but that is okay. Is this correct?

Reply:
Yes, I may look into those separately anyways.